src/components/inputs/date/ohri-date.component.tsx

```diff
@@ -129,9 +129,9 @@ const OHRIDate: React.FC<OHRIFormFieldProps> = ({ question, onChange, handler, p
   }, [encounterContext?.previousEncounter]);
 
   useEffect(() => {
-    getConceptNameAndUUID(question.questionOptions.concept).then((conceptTooltip) => {
-      setConceptName(conceptTooltip);
-    });
+    if (question.meta) {
+      setConceptName(question.meta?.display);
+    }
```

Comment:
@gitcliff do we still need the encompassing `useEffect`?

Reply:
oh i see 
i have removed it @samuelmale